packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -365,6 +375,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         {label && (
           <Label id={id + '-label'} disabled={disabled} required={required} htmlFor={id + '-input'} className={this._classNames.label}>
             {label}
+            <div className={this._classNames.screenReaderText}>{multiselectPlaceholder}</div>
```

Comment:
perhaps you meant `placeholder` here?

Reply:
I did not. This is only to help the multiselect string get read out.

Reply:
I'll add some comments or name it better.